tests/functional/RemoteWebDriverTest.php

```diff
@@ -245,4 +245,27 @@ public function testShouldSaveScreenshotToFile()
 
         unlink($screenshotPath);
     }
+
+    /**
+     * @covers ::takeScreenshot
+     */
+    public function testShouldSaveScreenshotToFileInSubDirectory()
+    {
+        if (!extension_loaded('gd')) {
+            $this->markTestSkipped('GD extension must be enabled');
+        }
+        if ($this->desiredCapabilities->getBrowserName() == WebDriverBrowserType::HTMLUNIT) {
+            $this->markTestSkipped('Screenshots are not supported by HtmlUnit browser');
+        }
+
+        $screenshotPath = sys_get_temp_dir() . '/fb-webdriver/dir/selenium-screenshot.png';
```

Comment:
To make sure the screenshot is actually saved into newly created directory, random directory and random file name could be used:

```suggestion
        $screenshotPath = sys_get_temp_dir() . '/' . uniqid('php-webdriver') . '/' . uniq('screenshot') '.png';
```